src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -201,7 +203,12 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
         return Path.join(directoryPath, fileName);
     }
 
-    public async CreatePreRequisiteParams(wizardInputs: WizardInputs): Promise<void> {
+    public async createPreRequisiteParams(wizardInputs: WizardInputs): Promise<void> {
+        // Create armAuthToken
+        wizardInputs.pipelineConfiguration.params["armAuthToken"] = "Bearer " + await GraphHelper.getAccessToken(wizardInputs.azureSession);
+    }
+
+    public async createACRResourceAndSPN( wizardInputs: WizardInputs): Promise<void> {
```

Comment:
Change the name of function to createSPN, ACR is specific to scenario.

Reply:
done